From m.nazarewicz@samsung.com  Thu Oct  7 11:07:28 2010
Date: Thu, 07 Oct 2010 13:05:24 +0200
From: Michal Nazarewicz <m.nazarewicz@samsung.com>
Subject: USB: gadget: file_storage: reuse definitions from a header file
To: Matthew Wilcox <willy@linux.intel.com>,
	Greg Kroah-Hartman <gregkh@suse.de>
Cc: linux-usb@vger.kernel.org, sarah.a.sharp@linux.intel.com,
	Michal Nazarewicz <mina86@mina86.com>,
	Alan Stern <stern@rowland.harvard.edu>,
	Matthew Wilcox <willy@linux.intel.com>,
	Greg Kroah-Hartman <gregkh@suse.de>
Message-id:  <afa94ac880ee460da05484972ba50ac9e76102a7.1286364302.git.mina86@mina86.com>

From: Michal Nazarewicz <mina86@mina86.com>

This commit changes storage_common.c and file_storage.c to
reuse definitions from linux/usb/storage.h header file.

Signed-off-by: Michal Nazarewicz <mina86@mina86.com>
Cc: Alan Stern <stern@rowland.harvard.edu>
Cc: Matthew Wilcox <willy@linux.intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/gadget/storage_common.c |   14 +-------------
 1 file changed, 1 insertion(+), 13 deletions(-)

--- a/drivers/usb/gadget/storage_common.c
+++ b/drivers/usb/gadget/storage_common.c
@@ -54,6 +54,7 @@
 
 
 #include <asm/unaligned.h>
+#include <linux/usb/storage.h>
 
 
 /*
@@ -156,19 +157,6 @@
 #define TYPE_DISK	0x00
 #define TYPE_CDROM	0x05
 
-/* USB protocol value = the transport method */
-#define USB_PR_CBI	0x00		/* Control/Bulk/Interrupt */
-#define USB_PR_CB	0x01		/* Control/Bulk w/o interrupt */
-#define USB_PR_BULK	0x50		/* Bulk-only */
-
-/* USB subclass value = the protocol encapsulation */
-#define USB_SC_RBC	0x01		/* Reduced Block Commands (flash) */
-#define USB_SC_8020	0x02		/* SFF-8020i, MMC-2, ATAPI (CD-ROM) */
-#define USB_SC_QIC	0x03		/* QIC-157 (tape) */
-#define USB_SC_UFI	0x04		/* UFI (floppy) */
-#define USB_SC_8070	0x05		/* SFF-8070i (removable) */
-#define USB_SC_SCSI	0x06		/* Transparent SCSI */
-
 /* Bulk-only data structures */
 
 /* Command Block Wrapper */
